ddtrace/contrib/aiohttp/middlewares.py

```diff
@@ -77,11 +101,17 @@ def on_prepare(request, response):
     request_span.finish()
 
 
-def trace_app(app, tracer, service='aiohttp-web'):
+def trace_app(app, tracer, service='aiohttp-web', enable_distributed=False):
```

Comment:
in that case no, since it configures an application object instead of patching it; anyway, the idea is to remove that too but in a later change